Userland/Libraries/LibGUI/Statusbar.cpp

```diff
@@ -38,6 +38,17 @@ NonnullRefPtr<Statusbar::Segment> Statusbar::create_segment()
     return widget;
 }
 
+void Statusbar::child_event(Core::ChildEvent& event)
+{
+    auto& event_to_forward = event;
+    if (event.type() == Event::ChildAdded && is<Widget>(*event.child()) && !event.insertion_before_child()) {
+        Core::ChildEvent modified_event(Event::ChildAdded, *event.child(), m_corner.ptr());
+        event_to_forward = modified_event;
```

Comment:
This is somewhat deceptive-looking, as we're actually *overwriting* the `event` object, not retargeting the `event_to_forward` reference to point at `modified_event`.